src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private Operation operation;
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void subtract(int quantity) {
+        this.quantity -= quantity;
+    }
+
+    public void add(int quantity) {
+        this.quantity += quantity;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String operation) {
+        this.operation = Operation.getOperationFromString(operation);
+    }
+
+    public enum Operation {
```

Comment:
If we are going to use enum outside of the enclosing class, it would be better to extract it to the separate file